src/page-content/content-nextjs.tsx

```diff
@@ -0,0 +1,168 @@
+import YoutubeEmbed from '../components/layouts/embed/embed'
+import { PropTypes as FeatureContentProps } from '../components/layouts/feature-content/feature-content'
+import { PropTypes as HeroImageProps } from '../components/layouts/hero/hero-image'
+import { PropTypes as SeoProps } from '../components/layouts/seo/seo'
+
+import Kratos from '../images/kratos/kratos.svg'
+
+const CheckCircle = (
+  <i className={'ph-check-circle-fill themed-primary size16'} />
+)
+const ArrowRight = <i className={'ph-arrow-right-bold size16'} />
+
+export const seo: SeoProps = {
+  description:
+    'NextJS Open source user management and identity server on Vercel',
+  title: 'Ory - Fast and reliable open source Auth for NextJS and Vercel.'
+}
+
+export const hero: HeroImageProps = {
+  id: 'nextjs.hero',
+  title: (
+    <>
+      No sweat authentication{' '}
+      <span className={'is-mute-text'}>for NextJS/React</span>
+    </>
+  ),
+  description: 'Start building now with a free Developer account.',
+  buttons: [
+    {
+      ctaTitle: 'Deploy',
+      style: 'filled',
+      to: 'https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fory%2Fkratos-nextjs-react-example&env=ORY_SDK_URL'
+    }
+  ],
+  image: (
+    <img
+      className="responsive"
+      loading="lazy"
+      alt="Examples of headless login and registrations forms with different styles using Ory Cloud."
+      src={Kratos}
+    />
+  )
+}
+
+export const featureContentVideo: FeatureContentProps = {
+  id: 'nextjs.featurecontent.video',
+  background: 'themed',
+  slant: true,
+  overline: 'Supports Vercel Edge Functions',
+  title: <>Lightning fast</>,
+  description: (
+    <>Onboard users without friction. Integrate with a few lines of code.</>
+  ),
+  content: <YoutubeEmbed embedId="ueEAoTQ-CTo" />,
+  button: {
+    ctaTitle: 'Get Started',
+    to: 'https://console.ory.sh/registration',
+    iconRight: ArrowRight
+  }
+}
+
+export const featureContentCodeBox: CodeBoxProps = {
+  tabs: [
+    {
+      filename: 'protected.tsx',
+      language: Languages.JavaScript,
+      code: `ory.toSession()
+  .then(({ data: session }) => {
+    console.log(session) // Our user session
+
+    return ory.createSelfServiceLogoutFlowUrlForBrowsers()
+      .then(({ data }) => {
+        setLogoutUrl(data.logout_url)
+      })
+  })
+`
+    },
+    {
+      filename: 'edge-function.ts',
+      language: Languages.JavaScript,
+      code: `//@ory/integrations for NextJS.
+import { config, createApiHandler } from '@ory/integrations/next-edge'  
+export { config }    
+// Create the Ory Cloud API "bridge".
+export default createApiHandler({
+fallbackToPlayground: true
+}) 
+`
+    },
+    {
+      filename: 'login.tsx',
+      language: Languages.JavaScript,
+      code: `//A few lines to implement a custom login form:
+ory.initializeSelfServiceLoginFlowForBrowsers()
+  .then(({ data }) => {
+    render(login)Form(data);
```

Comment:
This is invalid syntax, what I meant was to use login/registration/... as a word for each file...

```suggestion
    renderLoginForm(data);
```